_data/import-public.rb

```diff
@@ -1,20 +1,65 @@
 #! /usr/bin/env ruby
 #
 # Imports project data from the 18F Public Hub API.
-#
-# Author: Mike Bland (michael.bland@gsa.gov)
-# Date:   2014-12-22
 
 require 'open-uri'
 require 'json'
 
 DATA_DIR = File.dirname __FILE__
+TEAM_DIR = "_team"
 TEAM_API_BASE_URL = 'https://team-api.18f.gov/public/api/'
+TEAM_ENDPOINT = 'team'
+MEMBER_PROPERTIES = [
+  'city',
+  'first_name',
+  'full_name',
+  'github',
+  'last_name',
+  'name',
+  'role',
+  'state',
+  'team',
+  'twitter',
+  ].freeze
+
 
-['team'].each do |category|
-  category_data = (open("#{TEAM_API_BASE_URL}#{category}/").read)
-  results = JSON.parse(category_data)["results"]
-  open(File.join(DATA_DIR, "#{category}.json"), 'w') do |f|
+def update_team_file
+  results = JSON.parse(team_data)["results"]
+
+  open(File.join(DATA_DIR, "#{TEAM_ENDPOINT}.json"), 'w') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.